### DISCUSSION TITLE
【keyball44】トラックボールの上下動作が悪い

### DISCUSSION THREAD

**Discussion by Lee0313git** [Yowkees/keyball#702] [Q&A]
keyball44のトラックボール動作についてご相談させていただきます。
組み立て後、左右のカーソル移動は問題ないですが、上下の動きが悪いように感じております。

具体的には、上下にボールを移動させた際に、左右と同程度の移動を行わないような状態です。

こちらを解決する方法がございましたら教えていただけないでしょうか。

以下の点を試しております。

- Pro microの再はんだ
- トラックボール基板の光学センサIC再はんだ
- L字7Pコンスルーの再はんだ
- L字7Pコンスルー付近のジャンパ再はんだ
- Pro micro付近のジャンパ再はんだ
- L字7Pコンスルーのピンの曲がりチェック（おそらくまっすぐ？）
- ファームウェア再書き込み

※再はんだについては、全ての部分でコテを当てて再加熱を行うような形で実施
※ハンダをすべて取り除いての実施はしておりません

【補足】
- Pro microは遊舎工房様で購入
- 右側（トラックボール側）のみでも変わらず（左側を接続していない状態）
- 他キーの動作は問題なさそう

以上、よろしくお願いいたします。

【動画・画像】
https://github.com/user-attachments/assets/0b14d1f7-0a09-4058-9601-70ca0db09162

![20241206_184248](https://github.com/user-attachments/assets/6d393a43-4fee-4bba-944b-f95074ecaf64)
![20241206_184236](https://github.com/user-attachments/assets/812f95ec-5bc2-4374-8b4c-a5e4d0dff5c8)
![20241206_184159](https://github.com/user-attachments/assets/746aa359-f783-40e0-a7e3-b962901c1cc1)
![20241206_184154](https://github.com/user-attachments/assets/f3952ac1-2953-42de-bd27-ab0a0b2c317a)
![20241206_184123](https://github.com/user-attachments/assets/c4802c9e-8516-4101-9df3-107ced7b2235)
![20241206_184117](https://github.com/user-attachments/assets/4d4543cd-be88-4cf2-b82e-b9735bcca828)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

トラックボール基板がしっかりと7Pコンスルーに刺さっていない可能性が思い浮かびましたが、すでに何回か抜きさしされているようなので、関係なさそうです。
となると、センサーが何らかの理由で故障してしまったかも知れません。
新しい部品を試すなど、今後の検証方法をご相談したいので、下記白銀ラボお問い合わせよりご連絡いただければ幸いです。
https://shirogane-lab.net/inquiry

お手数ですがご対応お願いいたします。

**Reply by Lee0313git** [reply]
ご回答頂きましてありがとうございます。
センサー故障の可能性がありそうということですね。

お問合せ先かしこまりました。
上記へご連絡させていただきます。

引き続きどうかよろしくお願いいたします。
